src/routes/customer-routes.ts

```diff
@@ -6,3 +6,4 @@ export const router = express.Router()
 
 router.post('/upload', uploadCoi.single('file'), CustomerController.uploadPhoto)
 router.post('/company', CustomerController.createCustomerInfo)
+router.post('/desiredLanes', CustomerController.desiredLanesCreate)
```

Comment:
do not use camelcase inside endpoints, btw this endpoint should be removed

Reply:
done